cypress/objects/taskPage.js

```diff
@@ -0,0 +1,36 @@
+class TaskPage{
+    createTaskButtonClick(){
+        cy.contains('Create Task').click()
+    }
+
+    getTaskDetailsFormName(){
+        return cy.get('[id="taskDetailsForm.name"]')
+    }
+
+    getTaskDetailsFormDescription(){
+        return cy.get('[id="taskDetailsForm.description"]')
+    }
+
+    getTaskBillableDropDown(){
+        return cy.get('[aria-selected="true"]')
+    }
+
+    getYesFromDropDown(){
```

Comment:
this one could be parameterized in order to be able to select both - Yes and No in one method.